src/schema/users.ts

```diff
@@ -1185,6 +1223,25 @@ export const resolvers: IResolvers<any, Context> = {
         },
       );
     },
+    userIntegrations: async (
+      _,
+      __,
+      ctx: Context,
+    ): Promise<GQLUserIntegration[]> => {
+      const userIntegrations = await ctx.con
```

Comment:
Best practice would be to use graphorm to ensure graphql tree is resolved properly